app/uk/gov/hmrc/customs/declarations/metrics/controllers/CustomsDeclarationsMetricsController.scala

```diff
@@ -63,11 +63,12 @@ class CustomsDeclarationsMetricsController @Inject() (val logger: CdsLogger,
   }
 
   private def invalidJsonErrorResponse(jsError: JsError)(implicit messages: Messages): ErrorResponse = {
-    val contents = for {
+    val contents: Seq[ResponseContents] = (for {
       (jsPath, validationErrors) <- jsError.errors
```

Comment:
Can just call `.toSeq` here. `.toIndexedSeq` suggests we need constant-time random access - we probably don't need that for an error message in a HTTP response.